cmd/srcd/cmd/table.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+)
+
+type Table struct {
+	formats []string
+	header  []string
+	rows    [][]interface{}
+}
+
+func (t *Table) Header(header ...string) error {
```

Comment:
you don't check for this error in the code above that uses this method. Imo it would make API not so pretty if you do check. 
I would suggest to use `sticky error` pattern here. One of the examples of it is [bufio.Scanner](https://golang.org/pkg/bufio/#Scanner) from stdlib. But instead of defining `Err() error` method we can return it from `Print()`.

I have also found a very recent post from Dave Cheney explaining exactly this! https://dave.cheney.net/2019/01/27/eliminate-error-handling-by-eliminating-errors

Reply:
thanks for the tip!

Reply:
Addressed [here](https://github.com/src-d/engine/pull/409/commits/2a2e1f06b72c18c2130c3a678ca254f9dd7b1984).

Reply:
It doesn't implement the pattern but in this case most probably it doesn't matter anyway and actually simplifies the code 👍

Reply:
Yup, in this case, I think that this is enough :), but I'll surely remember this pattern for more complex things.